modules/imaging_browser/help/viewSession.md

```diff
@@ -1,58 +1,9 @@
 # Imaging Browser: View Session
 
-The View Session page displays a candidate's scans collected from
-a single timepoint.  General information on a candidate's session
-appears in a table at top including Patient Name, PSCID, DCCID,
-Visit Label, etc.
+This page displays a candidate's scans collected from a single timepoint. Click on any image to launch the BrainBrowser visualization tool. With every file displayed, there are different actions you can take, such as change the file's QC status, add or edit the QC comments, download the MINC file, or see a longitudinal view (in BrainBrowser). Click **Header Info** to see further scan parameters from the DICOM header.
 
-Files for the selected candidate's dataset are displayed below the
-general information table. Horizontal (axial), sagittal and coronal
-thumbnails are displayed -- click on any image to launch the
-BrainBrowser visualization tool.
+Each scan has a checkbox in the top left corner of its panel, to the left of its file name. You can check any number of these boxes, then click **3D + Overlay** which will overlay the selected volumes for your comparison. 
 
-To the right of the image thumbnails appears a list of scan parameters
-including QC Status, "Selected" (indicating which scan will be
```

Comment:
Add back the content from "QC status" to that end of sentence. (feel free to edit for style)

Reply:
( because these definitions are important)

Reply:
done

Reply:
awesome. thanks.